solidity/test/TestPhasedEscrow.js

```diff
@@ -174,82 +194,220 @@ describe("PhasedEscrow", () => {
+      )
+
+      rewardsBeneficiary = await BeaconRewardsEscrowBeneficiary.new(
+        token.address,
+        rewardsContract.address,
+        {from: owner}
+      )
+      await rewardsBeneficiary.transferOwnership(phasedEscrow.address, {
+        from: owner,
       })
 
-      it("emits a TokensWithdrawn event to the rewards beneficiary", async () => {
-        const receipt = await phasedEscrow.withdraw(transferAmount, {
-          from: owner,
-        })
+      await phasedEscrow.setBeneficiary(rewardsBeneficiary.address, {
+        from: owner,
+      })
+    })
 
-        expectEvent(receipt, "TokensWithdrawn", {
-          beneficiary: rewardsBeneficiary.address,
-          amount: web3.utils.toBN(transferAmount),
-        })
+    assertRewards(baseBalance, transferAmount)
+  })
+
+  describe("when withdrawing to a ECDSABackportRewardsEscrowBeneficiary", () => {
+    const baseBalance = 200000000
+    const transferAmount = 1800000
+
+    before(async () => {
+      rewardsContract = await TestECDSARewards.new(token.address)
+
+      rewardsBeneficiary = await ECDSABackportRewardsEscrowBeneficiary.new(
+        token.address,
+        rewardsContract.address,
+        {from: owner}
+      )
+      await rewardsBeneficiary.transferOwnership(phasedEscrow.address, {
+        from: owner,
       })
 
-      it("emits a RewardAdded event from the rewards beneficiary", async () => {
-        const receipt = resolveAllLogs(
-          (await phasedEscrow.withdraw(transferAmount, {from: owner})).receipt,
-          {curveRewards}
-        )
+      await phasedEscrow.setBeneficiary(rewardsBeneficiary.address, {
+        from: owner,
+      })
+    })
 
-        expectEvent(receipt, "RewardAdded", {
-          reward: web3.utils.toBN(transferAmount),
-        })
+    assertRewards(baseBalance, transferAmount)
+  })
+
+  describe("when withdrawing to a ECDSARewardsEscrowBeneficiary", () => {
+    const baseBalance = 200000000
+    const transferAmount = 178200000
+
+    before(async () => {
+      rewardsContract = await TestECDSARewards.new(token.address)
+
+      rewardsBeneficiary = await ECDSARewardsEscrowBeneficiary.new(
+        token.address,
+        rewardsContract.address,
+        {from: owner}
+      )
+      await rewardsBeneficiary.transferOwnership(phasedEscrow.address, {
+        from: owner,
+      })
+
+      await phasedEscrow.setBeneficiary(rewardsBeneficiary.address, {
+        from: owner,
       })
     })
+
+    assertRewards(baseBalance, transferAmount)
   })
+
+  async function assertRewards(baseBalance, transferAmount) {
+    it("withdraws specified tokens from escrow", async () => {
+      await phasedEscrow.withdraw(transferAmount, {from: owner})
+
+      expect(await token.balanceOf(phasedEscrow.address)).to.eq.BN(
+        baseBalance - transferAmount,
+        "Unexpected amount withdrawn"
+      )
+    })
+
+    it("transfers specified tokens to beacon backport rewards contract", async () => {
```

Comment:
```
    when withdrawing to a CurveRewardsEscrowBeneficiary
      ✓ withdraws specified tokens from escrow (291ms)
      ✓ transfers specified tokens to beacon backport rewards contract (287ms).  <--- NOPE
```

Reply:
Let's just drop `beacon backport` and we should be good.

Reply:
Done, removed.